src/main/java/seedu/address/logic/commands/HelpCommand.java

```diff
@@ -12,10 +12,20 @@ public class HelpCommand extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
             + "Example: " + COMMAND_WORD;
 
-    public static final String SHOWING_HELP_MESSAGE = "Opened help window.";
+    public static final String SHOWING_HELP_MESSAGE = "Here are the available commands and their usage:\n"
+            + "1. " + AddCommand.MESSAGE_USAGE + "\n"
+            + "2. " + EditCommand.MESSAGE_USAGE + "\n"
+            + "3. " + FindCommand.MESSAGE_USAGE + "\n"
+            + "4. " + ListCommand.MESSAGE_USAGE + "\n"
+            + "5. " + DeleteCommand.MESSAGE_USAGE + "\n"
+            + "6. " + ClearCommand.MESSAGE_USAGE + "\n"
+            + "7. " + ExitCommand.MESSAGE_USAGE + "\n"
+            + "For more detailed information on each command, please press F1 to refer to the project website.";
```

Comment:
Might be better if the usage messages are more brief or better formatted.  As of now the response box is very small (height-wise), so having a wall of text might affect user experience. You can also consider adding another `\n` to separate each numbered points with a blank line. 

For the record (for other members), this is how it looks like now:
![image](https://github.com/AY2324S2-CS2103T-T11-2/tp/assets/73763719/1f856072-bd3d-458a-a5dd-c540c3f450b5)


Since we already included the "for more info press F1" in the last line, we can consider keeping the messages simple.

Reply:
I can add \n to separate each numbered point.

Reply:
Hmm...then would it be better to not reuse the message_usage and instead create a separate message (displaying the more compressed details)?

But either way the current full message is still hard to read and looks very poorly formatted. Might need input from other people.